client/app/components/datagrid/UserOperatorDataGrid.tsx

```diff
@@ -0,0 +1,93 @@
+"use client";
```

Comment:
Moving this to it's own file was unnecessary but happened during my debugging that issue where the User Operator Datagrid would refresh every time you did anything with it. I kept this anyways since it will be necessary to have a client component to pass in actions once we do the server side pagination ticket for this table.